src/components/ReturnButton/ReturnButton.js

```diff
@@ -0,0 +1,9 @@
+import './ReturnButton.css'
+
+const ReturnButton = (props) => {
+  return (
+    <button className="returnButton" onClick={ props.deselect }>Go back!</button>
+  )
```

Comment:
Are you re-using this component? If not - it might be lighter for this to not be it's own component.